horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Yes, this idea to save the tabular-like style was not so good. Leading zero is a octal number and hex not good readable. But is a color code `#0088FF` like HTML not possible?
```
>>> print(030)
24
```